### PR TITLE
OCM-5422: Make the commit format more clear

### PR DESCRIPTION
**Pull request by nirarg**
**What this PR does / why we need it**:
Add more details about the TYPE options

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story (OCM-xxxx)*:
Fixes #[OCM-5422](https://issues.redhat.com//browse/OCM-5422)

**Change type**
- [ ] New feature
- [ ] Bug fix
- [ ] Build
- [x] CI
- [ ] Documentation
- [ ] Performance
- [ ] Refactor
- [ ] Style
- [ ] Unit tests
- [ ] Subsystem tests

**Checklist**
- [x] Subject and description added to both, commit and PR.
- [x] Relevant issues have been referenced.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
 <!--
 - Please ensure code changes are split into a series of logically independent commits.
 - Every commit should have a subject/title (What) and a description/body (Why).
+  The commit subject has the following format:
```
Comment (bardielle) [inline .github/pull_request_template.md:4]: fix it please

Comment (nirarg) [reply .github/pull_request_template.md]: Done

**Pull request merged**